bin/ch-pack-image

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+libexec=$(cd "$(dirname "$0")" && pwd)/../libexec/charliecloud
+. "${libexec}/base.sh"
+
+# shellcheck disable=SC2034
+usage=$(cat <<EOF
+Flatten a builder image into a Charliecloud packed file.
+
+Usage:
+
+  $ $(basename "$0") [-b BUILDER] [--fmt CH_PACK_FMT] IMAGE OUTDIR [ARGS ...]
+
+You must have sufficient privilege (via sudo) to run the Docker commands.
+EOF
+)
+
+parse_basic_args "$@"
+
+# Kludgey argument parsing.
+while true; do
+    case $1 in
+        -b|--builder)
+            shift
+            CH_BUILDER=$1
+            shift
+            ;;
+        -b=*|--builder=*)
```

Comment:
should this be `-b|--builder=*`?

Reply:
No, this should allow users to `-b=$BUILDER` or `--builder=$BUILDER`

Reply:
> No, this should allow users to `-b=$BUILDER` or `--builder=$BUILDER`

Do we do this anywhere else? If not, why start here?